Simplenote/Classes/String+Simplenote.swift

```diff
@@ -171,3 +171,10 @@ extension String {
         return ContentSlice(content: self, range: lowerBound..<upperBound, matches: matchingWordsRange)
     }
 }
+
+// MARK: Substring Instance Count
+extension String {
+    func occurancesOf(_ string: String) -> Int {
```

Comment:
Should it be `occurrences`? 🤔 

Interesting approach, too bad swift doesn't have a built in method for that 😞

Reply:
yes it should...  Spelling is not my greatest skill... fixed